docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+version: '3.8'
+
+services:
+  db:
+    image: mysql:8.0
+    container_name: todolist_db
+    environment:
+      MYSQL_ROOT_PASSWORD: root_password
+      MYSQL_DATABASE: todolist_db
+    volumes:
+      - todolist_db_data:/var/lib/mysql
+    networks:
+      - todolist_network
+
+  web:
+    build: .
+    container_name: todolist_web
+    command: ["sh", "-c", "python manage.py migrate && python manage.py runserver 0.0.0.0:8000"]
+    volumes:
+      - .:/app
+    ports:
+      - "8000:8000"
```

Comment:
The `ports` section maps port 8000 of the container to port 8000 on the host. However, the Dockerfile exposes port 8080. Ensure that the ports are consistent across the Dockerfile and `docker-compose.yml` to avoid connectivity issues. You may need to update the `EXPOSE` directive in the Dockerfile or the `ports` mapping here.